src/components/ClusterManager.js

```diff
@@ -363,6 +365,22 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
     this.resetUpdateInterval()
   }
 
+  componentDidUpdate(prevProps) {
+    const prevCluster = _.last(_.sortBy('createdDate', _.remove({ status: 'Deleting' }, prevProps.clusters))) || {}
+    const { status, jupyterUserScriptUri } = this.getCurrentCluster() || {}
+    if (status === 'Error' && prevCluster.status !== 'Error') {
+      notify('error', 'Error creating cluster', {
+        message: h(Fragment, [
+          strong(['Failed action: ']), jupyterUserScriptUri,
```

Comment:
It's an exceptional case, but clusters can enter Error status for other reasons, right? Is there a way to detect script errors vs other kinds of problems?